scripts/qa_crowbarsetup.sh

```diff
@@ -407,17 +407,23 @@ function addsp3testupdates()
 
 function addsp4testupdates()
 {
-    add_mount "SLES11-SP4-Updates" 'you.suse.de:/you/http/download/x86_64/update/SLE-SERVER/11-SP4/' "$tftpboot_repos_dir/SLES11-SP4-Updates/" "sp4tup"
+    add_mount "SLES11-SP4-Updates-test" \
+        'dist.suse.de:/dist/ibs/SUSE:/Maintenance:/Test:/SLE-SERVER:/11-SP4:/x86_64/update/' \
+        "$tftpboot_repos_dir/SLES11-SP4-Pool/" "sp4tup"
```

Comment:
It was intentional because we only pass Pool and Updates to the other nodes... but IMHO layering it over Updates would be better (provided that Updates-test is a superset of Updates)

or we could extend the autoyast.xml to have extra repos - which would miss the nodes using crowbar_join ... or some other mechanism to add repos via chef (I think there is already something in barclamp-provisioner)